src/main/java/org/folio/dcb/domain/mapper/DcbTransactionMapper.java

```diff
@@ -0,0 +1,39 @@
+package org.folio.dcb.domain.mapper;
+
+import org.folio.dcb.domain.entity.TransactionEntity;
+import org.folio.dcb.domain.dto.DcbTransaction;
+import org.folio.dcb.domain.dto.DcbPatron;
+import org.folio.dcb.domain.dto.DcbItem;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class DcbTransactionMapper {
```

Comment:
according to typical design, mappers consist of a single method.
I don't think, we should have a single class for multiple transformations

Reply:
I think we can use multiple methods in a mapper if it is dealing with same class.  In our case, In TransactionEntityMapper class, we have a method which convert dcbTransaction to TransactionEntity and in the new class , we are doing it vice versa. Since the purpose of both the classes are pretty much same, I feel we can combine 2 classes to one.